diagnostic.md

```diff
@@ -3,36 +3,44 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Low bandwith environments with single page applications.
+You can control state with API only development.
+Can do away with jquery.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system is handlebars. You load templates and components based on URLs.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+View
+Model
+View Model
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+A template is made up of components. Componets are pre-determined sub sets of the templates that you want to change.
 ```
 
 What is an opinionated framework? Is Ember opinionated? How or how not?
 
 ```sh
-# your answer here
+An opinionated framework is software that strictly guides the user to do things
```

Comment:
I would say that Ember requires developers, not users, to behave in certain ways. For example, it requires that files be named in particular ways, instead of allowing developers to choose. It favors convention over configuration.
.5/1